docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/02 - Projects.md

```diff
@@ -0,0 +1,33 @@
+# Projects
+
+## Overview
+The **Project** represents a study in LORIS and can have different subprojects (see [Subproject setup](04 - Subprojects.md) page) which, in turn, can be affiliated with different visits (see [Timepoints setup](05 - Timepoints.md) page). **Projects** are not required in LORIS but it is recommended to enable this feature when possible.
```

Comment:
I don't think the Markdown links are rendering properly.

Reply:
I think its due to the fact that the files dont actually exist ? locally they are working fine so I'm not sure, but I'll be able to fix these once I have multiple files merged